server/analyzer/init.go

```diff
@@ -22,29 +22,32 @@ import (
-	ruleId_AddDomainConstraintsToCasts                                // addDomainConstraintsToCasts
-	ruleId_AssignInsertCasts                                          // assignInsertCasts
-	ruleId_AssignUpdateCasts                                          // assignUpdateCasts
-	ruleId_ReplaceIndexedTables                                       // replaceIndexedTables
-	ruleId_ReplaceNode                                                // replaceNode
-	ruleId_ReplaceSerial                                              // replaceSerial
-	ruleId_AddImplicitPrefixLengths                                   // addImplicitPrefixLengths
-	ruleId_InsertContextRootFinalizer                                 // insertContextRootFinalizer
-	ruleId_ResolveType                                                // resolveType
-	ruleId_ReplaceArithmeticExpressions                               // replaceArithmeticExpressions
-	ruleId_OptimizeFunctions                                          // optimizeFunctions
-	ruleId_ValidateColumnDefaults                                     // validateColumnDefaults
-	ruleId_ValidateCreateTable                                        // validateCreateTable
-	rulesId_ResolveAlterColumn                                        // resolveAlterColumn
+	ruleId_TypeSanitizer                   analyzer.RuleId = iota + 1000 // typeSanitizer
+	ruleId_AddDomainConstraints                                          // addDomainConstraints
+	ruleId_AddDomainConstraintsToCasts                                   // addDomainConstraintsToCasts
+	ruleId_AssignInsertCasts                                             // assignInsertCasts
+	ruleId_AssignUpdateCasts                                             // assignUpdateCasts
+	ruleId_ConvertDropPrimaryKeyConstraint                               // convertDropPrimaryKeyConstraint
+	ruleId_GenerateForeignKeyName                                        // generateForeignKeyName
+	ruleId_ReplaceIndexedTables                                          // replaceIndexedTables
+	ruleId_ReplaceNode                                                   // replaceNode
+	ruleId_ReplaceSerial                                                 // replaceSerial
+	ruleId_AddImplicitPrefixLengths                                      // addImplicitPrefixLengths
+	ruleId_InsertContextRootFinalizer                                    // insertContextRootFinalizer
+	ruleId_ResolveType                                                   // resolveType
+	ruleId_ReplaceArithmeticExpressions                                  // replaceArithmeticExpressions
+	ruleId_OptimizeFunctions                                             // optimizeFunctions
+	ruleId_ValidateColumnDefaults                                        // validateColumnDefaults
+	ruleId_ValidateCreateTable                                           // validateCreateTable
+	rulesId_ResolveAlterColumn                                           // resolveAlterColumn
```

Comment:
I know it's not you, but we should rename `rulesId_ResolveAlterColumn` to `ruleId_ResolveAlterColumn`